tests/HelpMeCarry.tex

```diff
@@ -1,77 +1,137 @@
+
+  \item \textbf{Memorizing the operator:} The robot has to memorize the operator. During this phase, the robot may instruct the operator to follow a certain setup procedure.
+
+  \item \textbf{Following the operator:} When the robot signals that it is ready to start, the operator starts walking --in a natural way-- towards the car. Upon arrival, the operator will indicate the robot when they have reached their destination as instructed by the robot (e.g. by saying \quotes{here is the car} or \quotes{stop following me}).
+
+  \newcounter{enumTemp}
+  \setcounter{enumTemp}{\theenumi}
+  \item {[DSPL \& OPL]} \textbf{Bring the groceries in} \\
+  The robot is asked to deliver a bag with groceries to a specific location (e.g. \quotes{Take this bag to the kitchen table}).
+  \begin{enumerate}
+    \item \textbf{Bag pick-up:} The operator hands a bag to the robot to carry.
+
+    \item \textbf{Bag delivery:} The robot delivers the bag to the instructed destination. It may place the bag in the floor or onto the placement location.
+
+    \item \textbf{Asking for help:} Close to the delivery location is another person. The robot must face at them and kindly ask them to help carrying groceries into the house.
+  \end{enumerate}
+
+  \setcounter{enumi}{\theenumTemp}
+  \item {[SSPL only]} \textbf{Look for help} \\
+  The robot is asked to find a person in a given room and ask them to assist carrying the groceries (e.g. \quotes{Look for Louise in the Kitchen and ask her to help us}).
+  \begin{enumerate}
+    \item \textbf{Entering the house:} While on its way back to the house, the robot deals with different obstacles along it's path.
+    \begin{itemize}[leftmargin=3cm]
+      \item[\textbf{1st section}] While going back to the house, a person crosses robot's path.
+      \item[\textbf{2nd section}] While going back, a person steps in front of the robot and asks it for the time.
+    \end{itemize}
+
+    \item \textbf{Find a person:} After reaching the designated room, the robot needs to find a person (there is only one person in the room, the name is meaningless).
+  \end{enumerate}
+
+  \item \textbf{Memorizing the \emph{new} operator:} The robot has to memorize the operator that will help. During this phase, the robot may instruct the operator to follow a certain setup procedure.
+
+  \item \textbf{Guiding the operator:} When the robot signals that it is ready to start guiding, the robot guiding the operator to the car. The robot must clearly announce when the destination.
+  \begin{itemize}[leftmargin=3cm]
+    \item[DSPL \& OPL] \textbf{Closed door:} Along it's path to the car, the robot will find a closed door (most likely the entrance to the house) that will need to open to reach the destination.
```

Comment:
@kyordhel Just to clarify, a robot moves the same path twice from home to the car, right? If so, does this mean that at first the door is open and at the second time the it is closed and need to be open?